examples/pytorch/text-classification/Fine Tune BERT LLM.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        num_workers=3,  # Number of PyTorch workers to use.
</pre><p>Should we replace this with <code>1</code> aligned with the above comment:</p><p> </p><blockquote>At least&nbsp;<strong>1 GPU</strong>&nbsp;on your Kubernetes cluster to fine-tune BERT model on 1 worker.</blockquote><p> </p><p>Or, should we update the above comment?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/training-operator/pull/2021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Sure, let me change it to 3.